index.html

```diff
@@ -0,0 +1,21 @@
+Alejandro Patino Camargo
+
+About Me: 
```

Comment:
Not sure if the assignment said to use HTML tags but you should definitely try to get semantic in this file!

use a couple `h1`,`h2` tags for headers
`<p>` for your paragraphs
and `div`, `span`, `section` to help group content

Reply:
Hey Jeremy, I think I messed up somewhere in the process where I didn't push my final work. That's for the heads up and I'll fix it asap!

Reply:
all good!! No rush. If you need help figuring out what happened just let me know